Tests/Unit/Repository/PhpcrRepositoryTest.php

```diff
@@ -11,15 +11,16 @@
 
 namespace Symfony\Cmf\Component\Resource\Tests\Unit\Repository;
 
+use PHPCR\NodeType\NodeTypeInterface;
+use PHPCR\PathNotFoundException;
 use Symfony\Cmf\Component\Resource\Repository\PhpcrRepository;
+use Symfony\Cmf\Component\Resource\Repository\Resource\PhpcrResource;
 
 class PhpcrRepositoryTest extends RepositoryTestCase
 {
     public function setUp()
     {
-        $this->session = $this->prophesize('PHPCR\SessionInterface');
-        $this->finder = $this->prophesize('DTL\Glob\FinderInterface');
-        $this->node = $this->prophesize('PHPCR\NodeInterface');
+        parent::setUp();
         $this->child1 = $this->prophesize('PHPCR\NodeInterface');
         $this->child2 = $this->prophesize('PHPCR\NodeInterface');
```

Comment:
if `child1` and `child2` are required for this class, lets declare them in this class.